src/domain/schemas/document-schemas.js

```diff
@@ -52,3 +52,55 @@ export const hardDeleteSectionBodySchema = joi.object({
 export const hardDeleteDocumentBodySchema = joi.object({
   documentKey: idOrKeySchema.required()
 });
+
+const sectionDBSchema = joi.object({
+  revision: joi.string().required(),
+  key: idOrKeySchema.required(),
+  deletedOn: joi.date().allow(null).required(),
+  deletedBy: idOrKeySchema.allow(null).required(),
+  deletedBecause: joi.string().allow(null).required(),
+  type: joi.string().required(),
+  content: joi.alternatives().try(
+    joi.object().required(),
+    joi.any().allow(null).required()
+  ).required()
+});
+
+export const documentRevisionDBSchema = joi.object({
+  _id: idOrKeySchema.required(),
+  key: idOrKeySchema.required(),
+  order: joi.number().required(),
+  createdOn: joi.date().required(),
+  createdBy: idOrKeySchema.required(),
+  title: joi.string().required(),
+  slug: slugSchema,
+  language: joi.string().case('lower').required(),
+  sections: joi.array().items(sectionDBSchema).required(),
+  restoredFrom: joi.string().allow(null).allow('').required(),
```

Comment:
Is there a case where we have empty string? I think it should be either `null` or not-empty :thinking:

Reply:
Double check I shall. There might be cases in the DB already that will cause issues on import.

Reply:
Both `restoredFrom` and `originUrl` currently have in the DB empty string and `null` values.
So we either accept both, or we need to migrate both.
Due to the import feature, is not enough to ensure new documents consistently default these properties to `null`.

Reply:
To be honest, I would not do a migration at this point, though I would like the consistency I think we can live with this slight extra tolerance.

Reply:
Ok. If it's already inconsistent, we can fix it also later.